conf/zoo_sample.cfg

```diff
@@ -1,28 +1,29 @@
 # The number of milliseconds of each tick
 tickTime=2000
-# The number of ticks that the initial 
+# The number of ticks that the initial
 # synchronization phase can take
 initLimit=10
-# The number of ticks that can pass between 
+# The number of ticks that can pass between
 # sending a request and getting an acknowledgement
 syncLimit=5
 # the directory where the snapshot is stored.
-# do not use /tmp for storage, /tmp here is just 
-# example sakes.
-dataDir=/tmp/zookeeper
+# do not use /tmp for storage!!!.
+dataDir=/var/lib/zookeeper/data
```

Comment:
I am not sure this is so safe.

This is just an example file and it is only for dev/testing purpose.

Therefore it is not sure that an user/developer  is able to write into that directory

Reply:
Oh, I know. But some people use this config for their production environment. 
Is is awful ? Yes!  Maybe this config no good as example, but is good for operations engineer who are supported production system after `newbie DevOps` engineers. Maybe you have ideas on this?

Reply:
Look at an example from [kafka](https://github.com/apache/kafka/blob/trunk/config/server.properties#L60) which also use the `/tmp` as the default dir to store the logs

Reply:
@maoling I know it, it's look like 
![time bomb](https://i.imgur.com/EL1RN4A.gif)
* I had 2 production installation whith `log.dirs=/tmp/kafka-logs` in last year.

Reply:
![image](https://user-images.githubusercontent.com/11016631/54836486-4173f280-4cff-11e9-9360-c68aba7b893e.png)